.golangci.yml

```diff
@@ -6,6 +6,7 @@ linters:
     - interfacer # deprecated
     - maligned # deprecated
     - scopelint # deprecated
+    - depguard
```

Comment:
What's the issue with this linter?

Reply:
`depguard` relies on maintaining a whitelist of approved imports - would be useful in a more restrictive development environment, but for us it's just noise for every non-`github.com/udacity` library